packages/server/src/main.ts

```diff
@@ -7,13 +7,42 @@ const client = new PrismaClient();
 
 app.use(cors({ origin: process.env["WEB_ORIGIN"] }));
 
+// 通信テスト
 app.get("/", (_, res) => {
   res.json({ message: "success" });
 });
 
+// Userテーブルの作成
+app.post("/user", async (request, response) => {
```

Comment:
リクエストボディとレスポンスに型をつけておくとより良いですね。[zod](https://github.com/colinhacks/zod) などのライブラリを使えばきちんと悪意のあるデータが入っていないか確認できますが、まずはいったん型を付けるところから始めてみたらいかがでしょう。

```suggestion
type PostUserRequest = {
  name: string;
};

type PostUserResponse = {
  id: number;
  name: string;
};

app.post("/user", async (request, response) => {
  const requestBody: PostUserRequest = request.body;
  
  ~~~

  const responseBody: PostUserResponse = {
     id: createdUser.id,
     name: createdUser.name,    
  };
});
```